index.html

```diff
@@ -127,7 +127,7 @@ <h3 class="headline">Here's a time line of Dr. Borlaug's life:</h3>
             target="_blank"
              >
             Wikipedia Entry.</a
-          >
```

Comment:
kindly remove this unnecessary indentation to improve readability and reduce the file size.